.github/workflows/ci.yml

```diff
@@ -187,6 +187,10 @@ jobs:
         with:
           name: build-${{ matrix.name }}
           path: pack/*
+      - name: Trigger TPIL build
+        run: |
+          curl -u '${{ github.repository_owner }}:${{ secrets.PUSH_NIGHTLY_TOKEN }}' 'https://api.github.com/repos/leanprover/theorem_proving_in_lean4/dispatches' -X POST -d '{ "event_type": "build_event" }'
+        if: matrix.os == 'ubuntu-latest' && matrix.release
```

Comment:
```suggestion
        if: matrix.name == 'Linux release' && github.event_name == 'push' && github.repository == 'leanprover/lean4'
```

Reply:
Thanks, I assume `github.repository == 'leanprover/lean4'` stops it from happening in a fork?